### PR TITLE
Add uploader verified by service extraction

### PR DESCRIPTION
**Pull request by B0pol**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I have tested the API against [NewPipe](https://github.com/TeamNewPipe/NewPipe).
- [ ] I agree to create a pull request for [NewPipe](https://github.com/TeamNewPipe/NewPipe) as soon as possible to make it compatible with the changed API.

### CODE REVIEW

**Review by TobiGr — CHANGES_REQUESTED** [review]
Please implement the expectedVerified methods in the default*Tests. 
Btw. I assigned that issue to me, so most of that is implemented. You can have a look here: https://github.com/TobiGr/NewPipeExtractor/commit/b772d5dbfb0a6c120cebcbb4d54f5ada9191f594 ...

**Comment by B0pol** [comment]
Sorry, I did not notice that.
I think we should keep this PR, there is SoundCloud implementation, artist verified badge, extracted in comments and channel…
I will push the defaultTests soon and let you implement the NewPipe part then.

**Comment by TobiGr** [comment]
Yes, we should use your implementation. I did not have time to finish everything yet (notice the incomplete JDoc, some missing tests, etc.).
As I said, I won't have much time for NewPipe in the next weeks. You can use the changes from my commit and apply them to your PR.

**Comment by B0pol** [comment]
> Please implement the expectedVerified methods in the default*Tests.

Done.

**Review by TobiGr — APPROVED** [review]

File: extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/search/YoutubeSearchExtractorTest.java [hunk 1]
```diff
@@ -269,4 +273,41 @@ public void clarificationTest() throws Exception {
...
 
     }
+
+    public static class ChannelVerified extends DefaultSearchExtractorTest {
+        private static SearchExtractor extractor;
+        private static final String QUERY = "bbc";
+
+        @BeforeClass
+        public static void setUp() throws Exception {
+            NewPipe.init(DownloaderTestImpl.getInstance());
```
Comment (XiangRongLin) [inline extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/search/YoutubeSearchExtractorTest.java:283]: @B0pol Can do another PR where you replace this with the `DownloaderFactory`

**Pull request merged**